examples/rigid/single_franka_envs.py

```diff
@@ -11,6 +11,12 @@ def main():
     parser.add_argument("-s", "--sep", action="store_true", default=False)
     parser.add_argument("-r", "--record", action="store_true", default=False)
     parser.add_argument("-n", "--num_env", type=int, default=1)
+    parser.add_argument(
+        "--fps-tracker-alpha",
+        type=float,
+        default=0.95,
+        help="lambda value for exponential moving average of fps. 0 means no averaging, 1 is infinite averaging",
```

Comment:
a little bit weird it's called alpha but a lambda values.

`1 is infinite averaging` → 1 is not updating at all

Reply:
Sure :)

Reply:
addressed in 5fc8b92